portal-cdk/portal_cdk/portal_cdk_stack.py

```diff
@@ -166,6 +169,31 @@ def __init__(
         else:
             custom_domain_args = {}
 
+        error_bucket = s3.Bucket(
+            self,
+            "ErrorPages",
+            auto_delete_objects=True,
+            removal_policy=RemovalPolicy.DESTROY,
+            versioned=False,
+            block_public_access=s3.BlockPublicAccess(
+                block_public_acls=False,
+                block_public_policy=False,
+                ignore_public_acls=False,
+                restrict_public_buckets=False,
+            ),
+            public_read_access=True,
```

Comment:
I think we do want to block public access. If Cloudfront can't read from the bucket, we should set the bucket permissions such that Cloudfront, and only cloudfront, can read from the bucket.